packages/tasks/src/library-to-tasks.ts

```diff
@@ -8,7 +8,7 @@ import type { PipelineType } from "./pipelines";
  * This mapping is generated automatically by "python-api-export-tasks" action in huggingface/api-inference-community repo upon merge.
  * Ref: https://github.com/huggingface/api-inference-community/pull/158
  */
-export const LIBRARY_TASK_MAPPING_EXCLUDING_TRANSFORMERS: Partial<Record<ModelLibraryKey, PipelineType[]>> = {
+export const LIBRARY_TASK_MAPPING_EXCLUDING_TRANSFORMERS: Partial<Record<ModelLibraryKey, WidgetType[]>> = {
```

Comment:
i would remove `"conversational"` from this file and so we can keep `PipelineType` (i.e., https://huggingface.co/tasks/conversational can disappear)